flow/cmd/handler.go

```diff
@@ -81,8 +73,8 @@ func (h *FlowRequestHandler) createCdcJobEntry(ctx context.Context,
 		source_table_identifier, destination_table_identifier) VALUES ($1, $2, $3, $4, $5, $6, $7)
 		`, workflowID, req.ConnectionConfigs.FlowJobName, sourcePeerID, destinationPeerID,
 			"Mirror created via GRPC",
-			schemaForTableIdentifier(v.SourceTableIdentifier, sourePeerType),
```

Comment:
ooc, why did this get removed?

Reply:
updated PR description, but these columns no longer seem to be used (so are only informational) & `public.` only really makes sense for a couple databases